src/client.js

```diff
@@ -270,25 +271,43 @@ Client.prototype.input = function(data) {
 	var client = this;
 	var text = data.text.trim();
 	var target = client.find(data.target);
+
 	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+		text = "say " + text;
+	} else {
+		text = text.substr(1);
+
+		// If after removing first / we still have a /, send it as a message
+		if (text.charAt(0) === "/") {
+			text = "say " + text;
+		}
 	}
+
 	var args = text.split(" ");
-	var cmd = args.shift().replace("/", "").toLowerCase();
+	var cmd = args.shift().toLowerCase();
+	var result;
+
 	_.each(inputs, function(plugin) {
 		try {
 			var path = "./plugins/inputs/" + plugin;
 			var fn = require(path);
```

Comment:
Can we load the `require`s for all `input`s once and for all after we create the initial array L35?
Unless I am being mistaken, `require`ing all inputs at every command seems like an unnecessary overhead to me.